src/main/java/org/openpnp/vision/FluentCv.java

```diff
@@ -107,6 +107,26 @@
         nu.pattern.OpenCV.loadShared();
     }
 
+    public enum ColorSpace {
+        Gray(0),
+        Bgr(1),
+        Rgb(2),
+        Hls(3),
+        HlsFull(4),
+        Hsv(5),
+        HsvFull(6);
+        
+        private int code;
```

Comment:
Hi Tony

It seems this is obsolete, right?

Also, you could have used ordinal()

_Mark